css-two-column-layout/styles.css

```diff
@@ -0,0 +1,100 @@
+.container {
+  margin-left: auto;
+  margin-right: auto;
+  flex-wrap: wrap;
+  max-width: 1280px;
+  margin-top: 5rem;
+}
+
+.row {
+  display: flex;;
+}
+
+.column-full {
+  flex-basis: 100%;
+}
+
+.column-one-half {
+  flex-basis: 50%;
+  margin: 30px 30px 5px 30px;
+  padding: 13px 50px 0px 09px;
```

Comment:
`09px` can just be `9px`

Reply:
Thanks for all the feedback, Cody! Much appreciated :)